x/twap/types/utils.go

```diff
@@ -10,25 +10,6 @@ import (
 	"github.com/osmosis-labs/osmosis/v11/osmoutils"
 )
 
-func NewTwapRecord(k AmmInterface, ctx sdk.Context, poolId uint64, denom0 string, denom1 string) (TwapRecord, error) {
-	if !(denom0 > denom1) {
-		return TwapRecord{}, fmt.Errorf("precondition denom0 > denom1 not satisfied. denom0 %s | denom1 %s", denom0, denom1)
-	}
```

Comment:
If we remove this check here, I think we need to be clear in the spec about guarantees we expect. (Which methods expect correct order, which ones will swap order)

Reply:
Done!